pkg/features/kube_features.go

```diff
@@ -321,6 +328,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	GCERegionalPersistentDisk:                   {Default: true, PreRelease: utilfeature.Beta},
 	RunAsGroup:                                  {Default: false, PreRelease: utilfeature.Alpha},
 	VolumeSubpath:                               {Default: true, PreRelease: utilfeature.GA},
+	VolumeSubpathExpansion:                      {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
alpha features should be off by default

Reply:
Switched to false

Reply:
This now breaks the e2e tests though because it won't be enabled anymore